exercises/bank-account/canonical-data.json

```
@@ -0,0 +1,327 @@
+{
+  "exercise": "bank-account",
+  "comments": [
+    "This exercise is a good candidate to practice concurrency, which",
+    "may not be available or possible in the implementing language track.",
+    "In this case, all tests can be executed on a single thread. If there",
+    "are tests that don't work using a single thread, these should be",
+    "marked with a concurrency scenario."
+  ],
+  "cases": [
+    {
+      "uuid": "983a1528-4ceb-45e5-8257-8ce01aceb5ed",
+      "description": "Newly opened account has zero balance",
+      "property": "bankAccount",
+      "input": {
+        "operations": [
+          {
+            "operation": "open"
+          },
+          {
+            "operation": "amount",
+            "expected": 0
+          }
+        ]
+      },
+      "expected": {}
+    },
+    {
+      "uuid": "e88d4ec3-c6bf-4752-8e59-5046c44e3ba7",
+      "description": "Can deposit money",
+      "property": "bankAccount",
+      "input": {
+        "operations": [
+          {
+            "operation": "open"
+          },
+          {
+            "operation": "deposit",
+            "value": 100
+          },
+          {
+            "operation": "amount",
+            "expected": 100
+          }
+        ]
+      },
+      "expected": {}
+    },
+    {
+      "uuid": "3d9147d4-63f4-4844-8d2b-1fee2e9a2a0d",
+      "description": "Can deposit money sequentially",
+      "property": "bankAccount",
+      "input": {
+        "operations": [
+          {
+            "operation": "open"
+          },
+          {
+            "operation": "deposit",
+            "value": 100
+          },
+          {
+            "operation": "deposit",
+            "value": 50
+          },
+          {
+            "operation": "amount",
+            "expected": 150
+          }
+        ]
+      },
+      "expected": {}
+    },
+    {
+      "uuid": "08f1af07-27ae-4b38-aa19-770bde558064",
+      "description": "Can withdraw money",
+      "property": "bankAccount",
+      "input": {
+        "operations": [
+          {
+            "operation": "open"
+          },
+          {
+            "operation": "deposit",
+            "value": 100
+          },
+          {
+            "operation": "withdraw",
+            "value": 50
+          },
+          {
+            "operation": "amount",
+            "expected": 50
+          }
```

Let's make the withdraw amount different than 50, so people don't get confused about how to calculate the remaining balance
```suggestion
            "operation": "withdraw",
            "value": 75
          },
          {
            "operation": "amount",
            "expected": 25
          }
```